railties/lib/rails/generators/rails/scaffold/templates/scaffold.css

```diff
@@ -27,7 +39,8 @@ td {
   padding-right: 5px;
 }
 
-div.field, div.actions {
```

Comment:
I figured that'd be the most contentious change. Just fixed it though, to keep the element selector.